Sources/StreamChatUI/CommonViews/ContainerStackView.swift

```diff
@@ -496,13 +496,15 @@ extension UILayoutPriority {
 
 extension NSLayoutConstraint {
     func setTemporaryConstant(_ value: CGFloat) {
+        guard originalConstant == nil else { return }
```

Comment:
hm, the observer shouldn't even be invoked when there's an attempt to hide already hidden subview 🤔

```swift
hidingObserversByView[subview] = subview
            .observe(\.isHidden, options: [.new]) { [weak self] (view, isHiddenChange) in
                if isHiddenChange.newValue == true {
                    self?.hideArrangedSubview(view)
                } else {
                    self?.showArrangedSubview(view)
                }
            }
```

Reply:
That's what I would expect! But the test I added is failing without this fix 😞

Reply:
hm, how about
```swift
.observe(\.isHidden, options: [.new, .old]) { [weak self] (view, isHiddenChange) in
    guard isHiddenChange.oldValue != isHiddenChange.newValue else { return }
    ...
}
```

Reply:
I was testing this and I think the logic failed sometimes. 
The `show/hideArrangedSubview` has some logic inside, which may result in calling the `hideArrangedSubview` method, but not calling the `setTemporaryConstant` method inside. Therefore I fixed it on the lowest level possible to remove any other logic involved on the higher level